scripts/ci-equivalent-check.sh

```
@@ -0,0 +1,113 @@
+echo "════════════════════════════════════════════"
+echo "Summary"
+echo "════════════════════════════════════════════"
+for p in "${PASSED[@]}"; do printf "  ${GREEN}✓${NC} %s\n" "$p"; done
+for f in "${FAILED[@]}"; do printf "  ${RED}✗${NC} %s\n" "$f"; done
+echo ""
+echo "Passed: ${#PASSED[@]}"
+echo "Failed: ${#FAILED[@]}"
+
+if [[ ${#FAILED[@]} -gt 0 ]]; then
+  echo ""
+  printf "${RED}REFUSING TO PUSH — fix failures above first.${NC}\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)